touchforms/formplayer/static/formplayer/script/webformsession.js

```diff
@@ -326,9 +329,11 @@ WebFormSession.prototype.submitForm = function(form) {
 
     accumulate_answers = function(o) {
         if (ko.utils.unwrapObservable(o.type) !== 'question') {
-            $.each(o.children(), function(i, val) {
-                accumulate_answers(val);
-            });
+            if(typeof o.children == 'function') {
```

Comment:
what are you trying to check for here? are you seeing that children is not populated on a non question type? (i.e. repeat or group)